SPECS/cf-cli/cf-cli.spec

```diff
@@ -68,6 +69,9 @@ install -p -m 755 -t %{buildroot}%{_bindir} ./out/cf
 %{_bindir}/cf
 
 %changelog
+* Tue Apr 22 2025 Archana Shettigar <v-shettigara@microsoft.com> - 8.7.11-3
+- Fix CVE-2025-22872 with an upstream patch
```

Comment:
You can just say "patch CVE-2025-22872" as we are already providing the upstream patch details in the patch file itself.